test/regression/cases/idle/experiment.yaml

```diff
@@ -34,7 +34,7 @@ checks:
     description: "Memory usage quality gate. This puts a bound on the total agent memory usage."
     bounds:
       series: total_rss_bytes
-      upper_bound: "430.0 MiB"
+      upper_bound: "425.0 MiB"
```

Comment:
Sorry, late realization here. This should also be set in https://github.com/DataDog/datadog-agent/blob/main/test/regression/cases/quality_gate_idle/experiment.yaml. This file is being phased out in favor of the `quality_gate` prefixed one.